### ISSUE TITLE
Add more accessibilityIdentifier's

### ISSUE THREAD

**Issue by devinreams** [mozilla-lockbox/lockbox-ios#471]
From @isabelrios https://github.com/mozilla-lockbox/lockbox-ios/issues/453#issuecomment-393860114

>The accessibility Label part is fixed, almost all items have one, but not the accessibilityIdentifier. There are only a few. I added new ones in #303 but still there are more missing...and for some of them I would need devs help, not sure how to do it...

Good call-out, thanks @isabelrios. We can help get those added. (and by we I mean @sashei and maybe me if I did a little more reading)

### Welcome

- [x] "Get Started" = getStarted.button in #453
- [x] Learn More button = learnMore.button in #479

### Items List

- [x] A-Z / Recency sort button = sorting.button in #453
- [X] Settings gear button = settings.button in #453
- [x] Search/filter field = filterEntries.textField  in #479
- [ ] Clear filter button
- [x] Cancel filter button = cancelFilterTextEntry.button in #479
- [x] No entries learn more button = learnMore.button in #479

### Item Detail

- [ ] Web address label/row
- [ ] Username label/row
- [ ] Password label/row
- [x] Show/Hide password button = reveal.button in #453
- [x] Learn more button = learnEditEntry.button in #453

### Settings List

- [ ] Send Feedback button/row
- [ ] FAQ button/row
- [x] Send Usage Data switch toggle = sendUsageData.switch in #453
- [ ] FxA "Account" settings button/row
- [ ] Auto Lock button/row
  - [ ] Auto Lock options/rows in view
- [ ] Open Websites In button/row
  - [ ] Browser options/rows in view
- [x] Lock Now button/row = lockNow.button in #453

#### Account Settings

- [x] User name = userName.label in #453
- [x] Disconnect button = disconnectFirefoxLockbox.button in #479
  - [ ] Cancel confirm button
  - [ ] Disconnect confirm button

### Lock Screen

- [x] Unlock button = biometrics.button in #479

**Comment by sashei** [comment]
Hi @isabelrios !! I'm adding `accessibilityIdentifier`s to all of the buttons and text fields in the application; would you find them useful for things like labels or images as well?

**Comment by isabelrios** [comment]
@sashei Thanks!! Please see the ones I already added in PR #303* that work is done, although the PR has not landed yet, but at some point those ids will be ready. But feel free if you want to add them using a different name.
The more ids we have, the better, but to start with having them on buttons and text fields should be enough.
Thank you very much!

* Acc.IDs added there:
**Welcome screen:**
getStarted.button

**Entry List**
settings.button
sorting.button

**Settings List**
sendUsageData.switch
lockNow.button

**Account Setting**
userName.label
learnEditEntry.button

**Entry detailed view**
reveal.button

**Comment by isabelrios** [comment]
Sorry @sashei just remembered about the Settings, I'm using in the tests the label for each cell's staticText, but it would be better if there is an id for each Setting option to use instead of the label.  Thanks!

**Comment by devinreams** [comment]
I updated the original description above to include all buttons/labels I found on each view and where they've been added ☝️
